openslides/mediafiles/models.py

```diff
@@ -44,7 +44,8 @@ class Meta:
             ('can_see', 'Can see the list of files'),
             ('can_see_hidden', 'Can see hidden files'),
             ('can_upload', 'Can upload files'),
-            ('can_manage', 'Can manage files'))
+            ('can_manage', 'Can manage files'),
+            ('can_manage_logos', 'Can manage logos'))
```

Comment:
Ok yes. I will change the `check_view_permissions` for update and partial update if the config value to change is of type `logo`. If so, only users with `can_manage_logos` are allowed to change these config values.
I'll move the permission `can_manage_logos` also to the core. I thing we use this feature not only in the mediafile (only the logos are configured there) but this has effect on several places in OpenSlides. Also the mediafile won't get a dep of the core.